### PR TITLE
Siril: Update platform

### PR DESCRIPTION
**Pull request by lock042**
Change from 42 to 44

### CODE REVIEW

File: org.free_astro.siril.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.free_astro.siril",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "42",
+    "runtime-version": "44",
```
Comment (hfiguiere) [inline org.free_astro.siril.json:4]: build will fail. ffms2 tries to detect ffmpeg with an API that no longer exist.

Comment (lock042) [reply org.free_astro.siril.json:4]: Ouch ok.
What is the fix then?

Comment (hfiguiere) [reply org.free_astro.siril.json:4]: either building the right version of ffmpeg, or fixing ffms2. The former is probably the easiest given how fluid ffmpeg API are.

Comment (lock042) [reply org.free_astro.siril.json:4]: So I add a new module for the ffmpeg compilation?

**Comment by lock042** [comment]
@hfiguiere : Looks like I'm wrong :). If you know how to do it. Could you help please? :)

**Comment by lock042** [comment]
@hfiguiere : is it ok to use master for a module, but with a commit number?

File: org.free_astro.siril.json [hunk 2]
```diff
@@ -223,11 +223,67 @@
...
+				}
+			],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://github.com/FFMS/ffms2/archive/2.40.tar.gz",
-                    "sha256": "82e95662946f3d6e1b529eadbd72bed196adfbc41368b2d50493efce6e716320"
+                    "type": "git",
+                    "url": "https://github.com/FFMS/ffms2.git",
+                    "branch": "master",
```
Comment (hfiguiere) [inline org.free_astro.siril.json]:
```suggestion
```

never a branch.

Comment (hfiguiere) [reply org.free_astro.siril.json]: if that version works with ffmpeg 5 that is in the SDK maybe building ffmpeg can be avoided.

Comment (lock042) [reply org.free_astro.siril.json]: OK, but if I can't use a branch, how I can do?

Comment (hfiguiere) [reply org.free_astro.siril.json]: just use the commit.

**Comment by hfiguiere** [comment]
but in the end you didn't update the platform....

**Comment by lock042** [comment]
@hfiguiere : Sorry, my computer is dead and I work on GitHub IDE ;).

**Pull request merged**